v-next/hardhat-toolbox-viem/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nomicfoundation/hardhat-toolbox-viem",
-  "version": "4.0.0-next.17",
```

Comment:
With the released version being `4.0.0`, we bump the pre-release version to `5.0.0-next.17`. So the next major version, but we keep the pre-release number in line with all other packages i.e. `-next.17`.